pkg/controller/controller_test.go

```diff
@@ -20,18 +22,19 @@ import (
 	"github.com/atlassian/smith/pkg/resources/apitypes"
 	"github.com/atlassian/smith/pkg/speccheck"
 	"github.com/atlassian/smith/pkg/store"
-
-	"github.com/ash2k/stager"
 	scClientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
 	scFake "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/fake"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	core_v1 "k8s.io/api/core/v1"
+	crv1 "k8s.io/apiextensions-apiserver/examples/client-go/apis/cr/v1"
```

Comment:
We have a CRD in Smith repo - could you change to use it instead? See `examples/sleeper/client.go` `SleeperCrd()` function. It is used in integration tests. Can get an instance of CRD and set the desired Status on it.